scaife_viewer/wsgi.py

```diff
@@ -17,8 +17,10 @@ def setup():
     from . import cts
     # calling this will prime the cache in the master process. each fork
     # will inherit it. gunicorn --preload is required for this to work.
-    cts.TextInventory.load()
-    print("Loaded text inventory")
+    DEBUG = bool(int(os.environ.get("DEBUG", "0")))
```

Comment:
I don't believe Django has loaded the settings at this point yet.

Reply:
Sorry for the confusion; I left comments out of turn.

I had meant that the fallback value should match `settings.DEBUG`, and I'd prefer it be `1` to optimize for local dev.

In general, we've tried to set defaults in `settings.py` for local dev and then override as-needed in production.

Reply:
I just tested and I think we can safely do an import of Django settings (because of `os.environ.setdefault("DJANGO_SETTINGS_MODULE", "scaife_viewer.settings")`).

`from . import cts` will load Django (because `cts.collections` contains `from django.conf import settings`).